src/constants.js

```diff
@@ -0,0 +1,5 @@
+export const API_KEY = '8cac6dec66e09ab439c081b251304443'
```

Comment:
The API key should not be hardcoded into the project, consider creating a ".env" file to store this, this would also allow a separation of development and production keys.